Traceback (most recent call last):
  File "/home/ec2-user/autoflipp.online/flask-server/_AUDI_SEARCH.py", line 87, in <module>
    driver.get(url)
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 356, in get
    self.execute(Command.GET, {"url": url})
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/selenium/webdriver/remote/errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: timeout: Timed out receiving message from renderer: 299.152
  (Session info: chrome-headless-shell=126.0.6478.126)
Stacktrace:
#0 0x55d6bbc31c7a <unknown>
#1 0x55d6bb914e2c <unknown>
#2 0x55d6bb8fc1a6 <unknown>
#3 0x55d6bb8fbd85 <unknown>
#4 0x55d6bb8f9cc6 <unknown>
#5 0x55d6bb8fa33f <unknown>
#6 0x55d6bb90ac67 <unknown>
#7 0x55d6bb9211be <unknown>
#8 0x55d6bb9268ab <unknown>
#9 0x55d6bb8faa8a <unknown>
#10 0x55d6bb92103e <unknown>
#11 0x55d6bb9a3679 <unknown>
#12 0x55d6bb984363 <unknown>
#13 0x55d6bb954247 <unknown>
#14 0x55d6bb954b9e <unknown>
#15 0x55d6bbbf824b <unknown>
#16 0x55d6bbbfc2f1 <unknown>
#17 0x55d6bbbe3afe <unknown>
#18 0x55d6bbbfce52 <unknown>
#19 0x55d6bbbc879f <unknown>
#20 0x55d6bbc21638 <unknown>
#21 0x55d6bbc21810 <unknown>
#22 0x55d6bbc30dac <unknown>
#23 0x7f3ce389f7f2 start_thread

Traceback (most recent call last):
  File "/home/ec2-user/autoflipp.online/flask-server/_AUDI_SEARCH.py", line 87, in <module>
    driver.get(url)
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 356, in get
    self.execute(Command.GET, {"url": url})
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/selenium/webdriver/remote/errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: timeout: Timed out receiving message from renderer: 299.544
  (Session info: chrome-headless-shell=126.0.6478.126)
Stacktrace:
#0 0x5555b39c8c7a <unknown>
#1 0x5555b36abe2c <unknown>
#2 0x5555b36931a6 <unknown>
#3 0x5555b3692d85 <unknown>
#4 0x5555b3690cc6 <unknown>
#5 0x5555b369133f <unknown>
#6 0x5555b36a1c67 <unknown>
#7 0x5555b36b81be <unknown>
#8 0x5555b36bd8ab <unknown>
#9 0x5555b3691a8a <unknown>
#10 0x5555b36b803e <unknown>
#11 0x5555b373a679 <unknown>
#12 0x5555b371b363 <unknown>
#13 0x5555b36eb247 <unknown>
#14 0x5555b36ebb9e <unknown>
#15 0x5555b398f24b <unknown>
#16 0x5555b39932f1 <unknown>
#17 0x5555b397aafe <unknown>
#18 0x5555b3993e52 <unknown>
#19 0x5555b395f79f <unknown>
#20 0x5555b39b8638 <unknown>
#21 0x5555b39b8810 <unknown>
#22 0x5555b39c7dac <unknown>
#23 0x7fec9e89f7f2 start_thread

Traceback (most recent call last):
  File "/home/ec2-user/autoflipp.online/flask-server/_AUDI_SEARCH.py", line 21, in <module>
    driver = webdriver.Chrome(service=service, options=options)
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/selenium/webdriver/chrome/webdriver.py", line 45, in __init__
    super().__init__(
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/selenium/webdriver/chromium/webdriver.py", line 66, in __init__
    super().__init__(command_executor=executor, options=options)
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 208, in __init__
    self.start_session(capabilities)
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 292, in start_session
    response = self.execute(Command.NEW_SESSION, caps)["value"]
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/selenium/webdriver/remote/errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: crashed.
  (unknown error: DevToolsActivePort file doesn't exist)
  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)
Stacktrace:
#0 0x55efe227e4e3 <unknown>
#1 0x55efe1fadc76 <unknown>
#2 0x55efe1fd6d78 <unknown>
#3 0x55efe1fd3029 <unknown>
#4 0x55efe2011ccc <unknown>
#5 0x55efe201147f <unknown>
#6 0x55efe2008de3 <unknown>
#7 0x55efe1fde2dd <unknown>
#8 0x55efe1fdf34e <unknown>
#9 0x55efe223e3e4 <unknown>
#10 0x55efe22423d7 <unknown>
#11 0x55efe224cb20 <unknown>
#12 0x55efe2243023 <unknown>
#13 0x55efe22111aa <unknown>
#14 0x55efe22676b8 <unknown>
#15 0x55efe2267847 <unknown>
#16 0x55efe2277243 <unknown>
#17 0x7f715009f7f2 start_thread

Traceback (most recent call last):
  File "/home/ec2-user/autoflipp.online/flask-server/_AUDI_SEARCH.py", line 21, in <module>
    driver = webdriver.Chrome(service=service, options=options)
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/selenium/webdriver/chrome/webdriver.py", line 45, in __init__
    super().__init__(
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/selenium/webdriver/chromium/webdriver.py", line 66, in __init__
    super().__init__(command_executor=executor, options=options)
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 208, in __init__
    self.start_session(capabilities)
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 292, in start_session
    response = self.execute(Command.NEW_SESSION, caps)["value"]
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/selenium/webdriver/remote/errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.SessionNotCreatedException: Message: session not created: This version of ChromeDriver only supports Chrome version 114
Current browser version is 126.0.6478.126 with binary path /usr/bin/google-chrome
Stacktrace:
#0 0x5652632e64e3 <unknown>
#1 0x565263015c76 <unknown>
#2 0x56526304304a <unknown>
#3 0x56526303e4a1 <unknown>
#4 0x56526303b029 <unknown>
#5 0x565263079ccc <unknown>
#6 0x56526307947f <unknown>
#7 0x565263070de3 <unknown>
#8 0x5652630462dd <unknown>
#9 0x56526304734e <unknown>
#10 0x5652632a63e4 <unknown>
#11 0x5652632aa3d7 <unknown>
#12 0x5652632b4b20 <unknown>
#13 0x5652632ab023 <unknown>
#14 0x5652632791aa <unknown>
#15 0x5652632cf6b8 <unknown>
#16 0x5652632cf847 <unknown>
#17 0x5652632df243 <unknown>
#18 0x7f250f69f7f2 start_thread

Traceback (most recent call last):
  File "/home/ec2-user/autoflipp.online/flask-server/_AUDI_SEARCH.py", line 21, in <module>
    driver = webdriver.Chrome(service=service, options=options)
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/selenium/webdriver/chrome/webdriver.py", line 45, in __init__
    super().__init__(
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/selenium/webdriver/chromium/webdriver.py", line 66, in __init__
    super().__init__(command_executor=executor, options=options)
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 208, in __init__
    self.start_session(capabilities)
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 292, in start_session
    response = self.execute(Command.NEW_SESSION, caps)["value"]
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "/home/ec2-user/autoflipp.online/flask-server/venv/lib64/python3.9/site-packages/selenium/webdriver/remote/errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.SessionNotCreatedException: Message: session not created: This version of ChromeDriver only supports Chrome version 114
Current browser version is 126.0.6478.126 with binary path /usr/bin/google-chrome
Stacktrace:
#0 0x55f4093314e3 <unknown>
#1 0x55f409060c76 <unknown>
#2 0x55f40908e04a <unknown>
#3 0x55f4090894a1 <unknown>
#4 0x55f409086029 <unknown>
#5 0x55f4090c4ccc <unknown>
#6 0x55f4090c447f <unknown>
#7 0x55f4090bbde3 <unknown>
#8 0x55f4090912dd <unknown>
#9 0x55f40909234e <unknown>
#10 0x55f4092f13e4 <unknown>
#11 0x55f4092f53d7 <unknown>
#12 0x55f4092ffb20 <unknown>
#13 0x55f4092f6023 <unknown>
#14 0x55f4092c41aa <unknown>
#15 0x55f40931a6b8 <unknown>
#16 0x55f40931a847 <unknown>
#17 0x55f40932a243 <unknown>
#18 0x7f639169f7f2 start_thread

Button not found in this iframe, continuing to next iframe: Message: 
Stacktrace:
#0 0x563fe1b33c7a <unknown>
#1 0x563fe1816e2c <unknown>
#2 0x563fe1863661 <unknown>
#3 0x563fe1863751 <unknown>
#4 0x563fe18a7f64 <unknown>
#5 0x563fe18865ed <unknown>
#6 0x563fe18a5303 <unknown>
#7 0x563fe1886363 <unknown>
#8 0x563fe1856247 <unknown>
#9 0x563fe1856b9e <unknown>
#10 0x563fe1afa24b <unknown>
#11 0x563fe1afe2f1 <unknown>
#12 0x563fe1ae5afe <unknown>
#13 0x563fe1afee52 <unknown>
#14 0x563fe1aca79f <unknown>
#15 0x563fe1b23638 <unknown>
#16 0x563fe1b23810 <unknown>
#17 0x563fe1b32dac <unknown>
#18 0x7fe73309f7f2 start_thread

Cookie consent has been accepted.
Data from 5 pages saved to 'AUDI_SEARCH.json'
Button not found in this iframe, continuing to next iframe: Message: 
Stacktrace:
#0 0x5613f902cc7a <unknown>
#1 0x5613f8d0fe2c <unknown>
#2 0x5613f8d5c661 <unknown>
#3 0x5613f8d5c751 <unknown>
#4 0x5613f8da0f64 <unknown>
#5 0x5613f8d7f5ed <unknown>
#6 0x5613f8d9e303 <unknown>
#7 0x5613f8d7f363 <unknown>
#8 0x5613f8d4f247 <unknown>
#9 0x5613f8d4fb9e <unknown>
#10 0x5613f8ff324b <unknown>
#11 0x5613f8ff72f1 <unknown>
#12 0x5613f8fdeafe <unknown>
#13 0x5613f8ff7e52 <unknown>
#14 0x5613f8fc379f <unknown>
#15 0x5613f901c638 <unknown>
#16 0x5613f901c810 <unknown>
#17 0x5613f902bdac <unknown>
#18 0x7fee4d89f7f2 start_thread

Cookie consent has been accepted.
Data from 5 pages saved to 'AUDI_SEARCH.json'
Button not found in this iframe, continuing to next iframe: Message: 
Stacktrace:
#0 0x5603dec1bc7a <unknown>
#1 0x5603de8fee2c <unknown>
#2 0x5603de94b661 <unknown>
#3 0x5603de94b751 <unknown>
#4 0x5603de98ff64 <unknown>
#5 0x5603de96e5ed <unknown>
#6 0x5603de98d303 <unknown>
#7 0x5603de96e363 <unknown>
#8 0x5603de93e247 <unknown>
#9 0x5603de93eb9e <unknown>
#10 0x5603debe224b <unknown>
#11 0x5603debe62f1 <unknown>
#12 0x5603debcdafe <unknown>
#13 0x5603debe6e52 <unknown>
#14 0x5603debb279f <unknown>
#15 0x5603dec0b638 <unknown>
#16 0x5603dec0b810 <unknown>
#17 0x5603dec1adac <unknown>
#18 0x7feb8ec9f7f2 start_thread

Cookie consent has been accepted.
Traceback (most recent call last):
  File "/home/ec2-user/autoflipp.online/flask-server/_AUDI_SEARCH.py", line 102, in <module>
    with open('data/AUDI_SEARCH.json', 'w', encoding='utf-8') as json_file:
FileNotFoundError: [Errno 2] No such file or directory: 'data/AUDI_SEARCH.json'
Button not found in this iframe, continuing to next iframe: Message: 
Stacktrace:
#0 0x5633a18e8c7a <unknown>
#1 0x5633a15cbe2c <unknown>
#2 0x5633a1618661 <unknown>
#3 0x5633a1618751 <unknown>
#4 0x5633a165cf64 <unknown>
#5 0x5633a163b5ed <unknown>
#6 0x5633a165a303 <unknown>
#7 0x5633a163b363 <unknown>
#8 0x5633a160b247 <unknown>
#9 0x5633a160bb9e <unknown>
#10 0x5633a18af24b <unknown>
#11 0x5633a18b32f1 <unknown>
#12 0x5633a189aafe <unknown>
#13 0x5633a18b3e52 <unknown>
#14 0x5633a187f79f <unknown>
#15 0x5633a18d8638 <unknown>
#16 0x5633a18d8810 <unknown>
#17 0x5633a18e7dac <unknown>
#18 0x7f40fda9f7f2 start_thread

Cookie consent has been accepted.
Data from 5 pages saved to 'AUDI_SEARCH.json'
Button not found in this iframe, continuing to next iframe: Message: 
Stacktrace:
#0 0x564e2ceb9c7a <unknown>
#1 0x564e2cb9ce2c <unknown>
#2 0x564e2cbe9661 <unknown>
#3 0x564e2cbe9751 <unknown>
#4 0x564e2cc2df64 <unknown>
#5 0x564e2cc0c5ed <unknown>
#6 0x564e2cc2b303 <unknown>
#7 0x564e2cc0c363 <unknown>
#8 0x564e2cbdc247 <unknown>
#9 0x564e2cbdcb9e <unknown>
#10 0x564e2ce8024b <unknown>
#11 0x564e2ce842f1 <unknown>
#12 0x564e2ce6bafe <unknown>
#13 0x564e2ce84e52 <unknown>
#14 0x564e2ce5079f <unknown>
#15 0x564e2cea9638 <unknown>
#16 0x564e2cea9810 <unknown>
#17 0x564e2ceb8dac <unknown>
#18 0x7fe8ada9f7f2 start_thread

Cookie consent has been accepted.
Data from 5 pages saved to 'AUDI_SEARCH.json'
Button not found in this iframe, continuing to next iframe: Message: 
Stacktrace:
#0 0x55b5d2e1cc7a <unknown>
#1 0x55b5d2affe2c <unknown>
#2 0x55b5d2b4c661 <unknown>
#3 0x55b5d2b4c751 <unknown>
#4 0x55b5d2b90f64 <unknown>
#5 0x55b5d2b6f5ed <unknown>
#6 0x55b5d2b8e303 <unknown>
#7 0x55b5d2b6f363 <unknown>
#8 0x55b5d2b3f247 <unknown>
#9 0x55b5d2b3fb9e <unknown>
#10 0x55b5d2de324b <unknown>
#11 0x55b5d2de72f1 <unknown>
#12 0x55b5d2dceafe <unknown>
#13 0x55b5d2de7e52 <unknown>
#14 0x55b5d2db379f <unknown>
#15 0x55b5d2e0c638 <unknown>
#16 0x55b5d2e0c810 <unknown>
#17 0x55b5d2e1bdac <unknown>
#18 0x7f08ae49f7f2 start_thread

Cookie consent has been accepted.
Data from 5 pages saved to 'AUDI_SEARCH.json'
Button not found in this iframe, continuing to next iframe: Message: 
Stacktrace:
#0 0x5645e83e3c7a <unknown>
#1 0x5645e80c6e2c <unknown>
#2 0x5645e8113661 <unknown>
#3 0x5645e8113751 <unknown>
#4 0x5645e8157f64 <unknown>
#5 0x5645e81365ed <unknown>
#6 0x5645e8155303 <unknown>
#7 0x5645e8136363 <unknown>
#8 0x5645e8106247 <unknown>
#9 0x5645e8106b9e <unknown>
#10 0x5645e83aa24b <unknown>
#11 0x5645e83ae2f1 <unknown>
#12 0x5645e8395afe <unknown>
#13 0x5645e83aee52 <unknown>
#14 0x5645e837a79f <unknown>
#15 0x5645e83d3638 <unknown>
#16 0x5645e83d3810 <unknown>
#17 0x5645e83e2dac <unknown>
#18 0x7fc588c9f7f2 start_thread

Cookie consent has been accepted.
Data from 5 pages saved to 'AUDI_SEARCH.json'
Button not found in this iframe, continuing to next iframe: Message: 
Stacktrace:
#0 0x560f8c20ec7a <unknown>
#1 0x560f8bef1e2c <unknown>
#2 0x560f8bf3e661 <unknown>
#3 0x560f8bf3e751 <unknown>
#4 0x560f8bf82f64 <unknown>
#5 0x560f8bf615ed <unknown>
#6 0x560f8bf80303 <unknown>
#7 0x560f8bf61363 <unknown>
#8 0x560f8bf31247 <unknown>
#9 0x560f8bf31b9e <unknown>
#10 0x560f8c1d524b <unknown>
#11 0x560f8c1d92f1 <unknown>
#12 0x560f8c1c0afe <unknown>
#13 0x560f8c1d9e52 <unknown>
#14 0x560f8c1a579f <unknown>
#15 0x560f8c1fe638 <unknown>
#16 0x560f8c1fe810 <unknown>
#17 0x560f8c20ddac <unknown>
#18 0x7fc66d29f7f2 start_thread

Cookie consent has been accepted.
Data from 5 pages saved to 'AUDI_SEARCH.json'
Button not found in this iframe, continuing to next iframe: Message: 
Stacktrace:
#0 0x5566de46bc7a <unknown>
#1 0x5566de14ee2c <unknown>
#2 0x5566de19b661 <unknown>
#3 0x5566de19b751 <unknown>
#4 0x5566de1dff64 <unknown>
#5 0x5566de1be5ed <unknown>
#6 0x5566de1dd303 <unknown>
#7 0x5566de1be363 <unknown>
#8 0x5566de18e247 <unknown>
#9 0x5566de18eb9e <unknown>
#10 0x5566de43224b <unknown>
#11 0x5566de4362f1 <unknown>
#12 0x5566de41dafe <unknown>
#13 0x5566de436e52 <unknown>
#14 0x5566de40279f <unknown>
#15 0x5566de45b638 <unknown>
#16 0x5566de45b810 <unknown>
#17 0x5566de46adac <unknown>
#18 0x7f751cc9f7f2 start_thread

Cookie consent has been accepted.
Data from 5 pages saved to 'AUDI_SEARCH.json'
